data/SYNTH/data.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Synth",
+  "symbol": "SYNTH",
+  "decimals": 18,
+  "description": "Multi-strategy crypto hedge fund",
+  "tokens": {
+    "ethereum": {
+      "address": "0xd41f3d112cb8695c7a8992e4055bd273f3ce8729"
+    },
+    "optimism": {
+      "address": "0xa925f4057d6E6C8FAf8bdE537Ad14BA91A1D0337"
```

Comment:
your json is malformed, so tests are failing; you're missing a closing `}` after this line, and see if you can validate this locally!